vcpkg/github-integration.md

```diff
@@ -52,7 +52,7 @@ You must enable the GitHub dependency graph in your repository's settings (enabl
 
 The following are known limitations in the current implementation. Even though we know of these, if they affect the value for you, we'd still like to hear from you so that we can prioritize what we need to fix and how we should fix it.
 
-* The version of vcpkg that is bundled with GitHub Actions runners does not have this feature yet. Ensure you are using a version of vcpkg that supports this feature.
+* The version of vcpkg that is bundled with GitHub Actions runners does not have this feature yet. Ensure you are using vcpkg with [commit SHA 8f3e8d6](https://github.com/microsoft/vcpkg/commit/8f3e8d6e95c7f91edea60c47a7d1414ab5ca9492) or later (merged on 2023-06-26).
```

Comment:
In the future we want to standardize how we communicate the minimum version required for specific features. Probably by using the vcpkg-tool release tag.

Reply:
Yeah, I think this is fine for now. And I agree, we should come up with some standard phrasing. I'd argue strongly however that using the vcpkg-tool release tag is not the right phrasing though, as users do not have an easy way use that information in a way that they can take action on. See my draft PR that would [enable users to bootstrap a particular version of vcpkg-tool](https://github.com/microsoft/vcpkg/pull/32156) that would make this easier though. As it stands now, it the vcpkg-tool tag gives the users a breadcrumb and expects them to follow it instead of giving them the bread directly.